src/Sulu/Bundle/ContentBundle/.gitignore

```diff
@@ -3,6 +3,7 @@
 /composer.phar
 /composer.lock
 /.sass-cache/
+/Resources/app/cache
```

Comment:
I would go even further and say we should remove the general rule from the root `.gitignore`, and leave the bundle specific ignores in this file. This way it would also work if we make a subtree split one day.